### PR TITLE
feat: Add AI Experimentation Solution

### PR DESCRIPTION
**Pull request by cartyc**
This package includes a project that is pre-configured with the required services to begin using vertex ai for experimentation with Google AI services.

Some services are not fully available in Canada and this project adds an organization policy exemption for the target project to facilitate the deployment. This package is not intended for use beyond profile 1 and the experimentation Landing Zone.

A future package will be added to demonstrate the creation of vertex ai components  using KRM.

### CODE REVIEW

**Review by fmichaelobrien — DISMISSED** [review]
Lgtm

**Review by fmichaelobrien — APPROVED** [review]
LGTM

File: solutions/vertexai/ai-experimentation/setters.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  #
+  #
+  #
+  ##########################
+  # Client
+  ##########################
+  #
+  # Name for the client, lowercase only
+  client-name: 'client1'
+  # id of the client management project created in client-landing-zone
```
Comment (davelanglois-ssc) [inline solutions/vertexai/ai-experimentation/setters.yaml:49]: this line seems not necessary

**Review by davelanglois-ssc — COMMENTED** [review]
very interesting use case

File: solutions/vertexai/ai-experimentation/setters.yaml (hunk 1 above)
Comment (cartyc) [reply solutions/vertexai/ai-experimentation/setters.yaml:49]: That seems fair! I believe I can remove that without complications.

**Comment by cartyc** [comment]
@davelanglois-ssc we've been seeing an influx of requests for GenAI sandboxes and the intent of this is to help expedite teams time to getting a PoC off the ground. Hopefully we can also leverage some of the VertexAI [Alphav1](https://github.com/GoogleCloudPlatform/k8s-config-connector/tree/master/crds) CRDs as well

**Review by fmichaelobrien — DISMISSED** [review]
LGTM - post comments/edits

**Review by fmichaelobrien — DISMISSED** [review]
LGTM
the autopilot readme code - I will test this later

**Review by fmichaelobrien — APPROVED** [review]
LGTM 
if you are still testing changes - I'll review them in batch later

**Comment by cartyc** [comment]
@fmichaelobrien ya I probably should put it as draft. I originally tested it with the experimenation landing zone and now running it with the just the client-landing-zone and started getting some errors. Hoping to allow it to work with both so users have some flexibility with deployment.

**Comment by cartyc** [comment]
Redeployed and working a-ok. Safe for a review :)

File: solutions/vertexai/ai-experimentation/Kptfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: ai-sandbox-project
+  annotations:
+    config.kubernetes.io/local-config: "true"
+info:
+  description: |
+    Landing zone v2 subpackage.
+    Depends on `experimentation/core-landing-zone`.
```
Comment (davelanglois-ssc) [inline solutions/vertexai/ai-experimentation/Kptfile:10]: I think this should read experimentation/client-landing-zone .... like in the README.md

File: solutions/vertexai/ai-experimentation/project-iam.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  namespace: projects # kpt-set: ${namespace}
+spec:
+  member: group:aisandbox-admins.domain.com # kpt-set: ${user-group}
+  role: roles/contentwarehouse.admin
+  resourceRef:
+    kind: Project
+    namespace: projects # kpt-set: ${namespace}
+    name: project-id # kpt-set: ${project-id}
+---
+## Document Warehouse Admin
```
Comment (davelanglois-ssc) [inline solutions/vertexai/ai-experimentation/project-iam.yaml:44]: comment not correct.. role is Document AI Admin

**Review by davelanglois-ssc — CHANGES_REQUESTED** [review]
minor updates to some comments,, other than that, it is a great addition !! good job chris

**Review by davelanglois-ssc — APPROVED** [review]

**Review by fmichaelobrien — APPROVED** [review]
LGTM

**Pull request merged**